packages/solid/src/reactive/signal.ts

```diff
@@ -1750,7 +1750,7 @@ type TODO = any;
  *
  * * If the error is thrown again inside the error handler, it will trigger the next available parent handler
  *
- * @description https://www.solidjs.com/docs/latest/api#onerror
+ * @description https://www.solidjs.com/docs/latest/api#onerror | https://docs.solidjs.com/reference/reactive-utilities/catch-error
```

Comment:
Just out of curiosity, Is there a reason why we kept the previous one here?

Reply:
as far as I can see, `onError` has not been taken to the new docs because the api is deprecated. So I left the old link and added the link to the recommended alternative